Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,94 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+from mentii import user_ctrl
+
+class MentiiAuthentication:
+
+  def __init__(self):
+    return
+
+  @staticmethod
+  def build_user_object(userData):
+    '''
+    Builds a standard user object to return to the client for all authenticated calls
+    '''
+    return {'email': userData['email']}
+
+  @staticmethod
+  def get_app_secret():
+    '''
+    Reads from the prodConfig.ini file for the appSecret
+    '''
+    parser = cp.ConfigParser()
+    parser.read("/config/prodConfig.ini")
+    return parser.get('MentiiAuthentication', 'appSecret')
```

Comment:
I'll add some error checking but I agree. We should create a story to create a module that can be imported to retrieve stuff from the config file. I won't be adding it because that alone is cleanup that wouldn't be in the scope of this story